rust/processor/src/bq_analytics/gcs_handler.rs

```diff
@@ -65,7 +68,16 @@ pub async fn upload_parquet_to_gcs(
 
         match upload_result {
             Ok(Ok(result)) => {
-                info!("File uploaded successfully to GCS: {}", result.name);
+                info!(
+                    table_name = table_name,
+                    uploaded_time_iso = now.to_rfc3339(),
+                    "File uploaded successfully to GCS: {}",
+                    result.name
```

Comment:
file_name = result.name,
"File uploaded successfully to GCS"

With this, you can search file name easily.

Reply:
good call!